bundle/manifests/multicluster-gateway-controller.clusterserviceversion.yaml

```diff
@@ -0,0 +1,599 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[]'
+    capabilities: Basic Install
+    createdAt: "2023-09-08T10:20:25Z"
+    operators.operatorframework.io/builder: operator-sdk-v1.28.0
+    operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
+  name: multicluster-gateway-controller.v0.0.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - kind: DNSHealthCheckProbe
+      name: dnshealthcheckprobes.kuadrant.io
+      version: v1alpha1
+    - kind: DNSPolicy
+      name: dnspolicies.kuadrant.io
+      version: v1alpha1
+    - kind: DNSRecord
+      name: dnsrecords.kuadrant.io
+      version: v1alpha1
+    - kind: ManagedZone
+      name: managedzones.kuadrant.io
+      version: v1alpha1
+    - kind: TLSPolicy
+      name: tlspolicies.kuadrant.io
+      version: v1alpha1
+  description: test
```

Comment:
should we update this?

Reply:
Ooops thought i had gone back and updated those